buildSrc/src/main/kotlin/com/datadog/gradle/config/AndroidConfig.kt

```diff
@@ -17,7 +17,7 @@ object AndroidConfig {
     const val MIN_SDK_FOR_COMPOSE = 21
     const val BUILD_TOOLS_VERSION = "31.0.0"
 
-    val VERSION = Version(1, 12, 0, Version.Type.Dev)
```

Comment:
We haven't started a `release/1.12.0` branch yet, so you shouldn't upgrade this

Reply:
Oh sorry this is a left over from my tests